app/services/reporting/active_users_count_report.rb

```diff
@@ -0,0 +1,109 @@
+module Reporting
+  class ActiveUsersCountReport
+    attr_reader :report_date
+
+    def initialize(report_date = Time.zone.today)
+      @report_date = report_date
+    end
+
+    def active_users_count_emailable_report
+      EmailableReport.new(
+        title: 'Active Users',
+        table: generate_report,
+        filename: 'active_users_count',
+      )
+    end
+
+    def generate_report
+      [
+        ['Active Users', 'IAL1', 'IDV', 'Total', 'Range start', 'Range end'],
+        monthly_report,
+        fiscal_report,
+      ]
+    end
+
+    private
+
+    def monthly_report
```

Comment:
maybe rename this `current_month_row`?
```suggestion
    def current_month_row
```